koordinates/exports.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+
+"""
+koordinates.exports
+===================
+
+The `Exports API <https://support.koordinates.com/hc/en-us/articles/208580966>`_
+provides an interface to create exports and download data from a Koordinates site.
+"""
+import logging
+
+from . import base
+from . import exceptions
+from .utils import is_bound
+
+
+logger = logging.getLogger(__name__)
+
+
+class CropLayerManager(base.Manager):
+    """
+    Accessor for querying Crop Layers.
+
+    Access via the ``exports.croplayers`` property of a :py:class:`koordinates.client.Client` instance.
+    """
+
+    _URL_KEY = 'CROPLAYER'
+
+    def get_feature(self, croplayer_id, cropfeature_id):
+        """
+        Gets a crop feature
+
+        :param int croplayer_id: ID of a cropping layer
+        :param int cropfeature_id: ID of a cropping feature
+        :rtype: CropFeature
+        """
+        target_url = self.client.get_url('CROPFEATURE', 'GET', 'single', {'croplayer_id': croplayer_id, 'cropfeature_id': cropfeature_id})
+        return self.client.get_manager(CropFeature)._get(target_url)
+
+
+class CropLayer(base.Model):
+    """
+    A crop layer provides features that can be used to crop exports to a geographic extent.
+    """
+    class Meta:
+        manager = CropLayerManager
+        relations = {
+            'features': ['CropFeature'],
+        }
+
+    @is_bound
+    def get_feature(self, cropfeature_id):
+        """
+        Gets a crop feature
+
+        :param int cropfeature_id: ID of a cropping feature
+        :rtype: CropFeature
+        """
+        return self._manager.get_feature(self.id, cropfeature_id)
+
+
+class CropFeatureManager(base.Manager):
+    """
+    Accessor for querying Crop Features.
+    """
+
+    _URL_KEY = 'CROPFEATURE'
+
+
+class CropFeature(base.Model):
+    """
+    A crop feature provides complex pre-defined geographic extents for cropping
+    and clipping Exports.
+    """
+    class Meta:
+        manager = CropFeatureManager
+        relations = {
+            'layer': CropLayer,
+        }
+
+    def _serialize(self):
+        return self.url
+
+
+class ExportManager(base.Manager):
+    """
+    Accessor for querying and creating Exports.
+
+    Access via the ``exports`` property of a :py:class:`koordinates.client.Client` instance.
+    """
+
+    _URL_KEY = 'EXPORT'
+    _options_cache = None
+
+    @property
+    def croplayers(self):
```

Comment:
Makes the 'crop' bit sound like a verb though, as if it's specifying 'which layers should I crop' as opposed to 'which layers should I use to crop my export'. Also it's not snake-cased in the API URL (and would look weird if it was) so I'd tend toward keeping it the same, ie `croplayers`